app/controllers/katello/api/v2/content_exports_controller.rb

```diff
@@ -50,23 +56,35 @@ def version
                                                "no greater than the specified size in gigabytes."), :required => false
     param :fail_on_missing_content, :bool, :desc => N_("Fails if any of the repositories belonging to this organization"\
                                                          " are unexportable. False by default."), :required => false
+    param :format, ::Katello::Pulp3::ContentViewVersion::Export::FORMATS,
+                   :desc => N_("Export formats. Choose syncable if content is to be imported via repository sync. "\
+                               "Choose importable if content is to be imported via hammer content-import.
+                                Defaults to importable."),
+                   :required => false
     def library
       tasks = async_task(::Actions::Pulp3::Orchestration::ContentViewVersion::ExportLibrary,
                           @organization,
                           destination_server: params[:destination_server],
                           chunk_size: params[:chunk_size_gb],
-                          fail_on_missing_content: ::Foreman::Cast.to_bool(params[:fail_on_missing_content]))
+                          fail_on_missing_content: ::Foreman::Cast.to_bool(params[:fail_on_missing_content]),
+                          format: find_export_format)
       respond_for_async :resource => tasks
     end
 
     api :POST, "/content_exports/repository", N_("Performs a full-export of the repository in library.")
     param :id, :number, :desc => N_("Repository identifier"), :required => true
     param :chunk_size_gb, :number, :desc => N_("Split the exported content into archives "\
                                                "no greater than the specified size in gigabytes."), :required => false
+    param :format, ::Katello::Pulp3::ContentViewVersion::Export::FORMATS,
+                   :desc => N_("Export formats. Choose syncable if content is to be imported via repository sync. "\
+                               "Choose importable if content is to be imported via hammer content-import.
+                                Defaults to importable."),
```

Comment:
(It's OK if the answer is "no", I'm not sure if you already evaluated some tradeoff here)

Reply:
I originally came here from the hammer-cli-katello PR trying to figure out where `--help` text is generated, thinking users might be curious to know what is actually different about the thing beyond what next steps you take with it. But I can't think of any way to do that without making it way too wordy -- seems like foreman-documentation is a better fit.

It also got me thinking that maybe this API description should be more API-centric to be neutral towards hammer/FAM. But then it would have to be overwritten to show something hammer-specific to hammer users. (FAM already has its own documentation, I guess only hammer just re-uses the descriptions from here)

Probably 99% of the users who read this would have gotten here from `hammer content-export --help` anyway so it's likely not worth the extra effort. With that perspective, I think this all looks great as-is :+1:

Reply:
> This description text is written out three separate times. Is there any sufficiently good strategy for avoiding the duplication, to reduce the diffs and make it easier to keep consistency in the future?

Thats a good point we may be able to dry it up by creating a String constant somewhere and copy pasting that 3 times. But usually I dont like to over optimize on API doc comments. Reason being if we wanted to change message from ` Choose syncable if content is to be imported` to  `Choose syncable if content is to be imported into a repository` etc I should be able to do that independently.

Reply:
> I originally came here from the hammer-cli-katello PR trying to figure out where `--help` text is generated, thinking users might be curious to know what is actually different about the thing beyond what next steps you take with it. But I can't think of any way to do that without making it way too wordy -- seems like foreman-documentation is a better fit.
> 
> It also got me thinking that maybe this API description should be more API-centric to be neutral towards hammer/FAM. But then it would have to be overwritten to show something hammer-specific to hammer users. (FAM already has its own documentation, I guess only hammer just re-uses the descriptions from here)
> 
> Probably 99% of the users who read this would have gotten here from `hammer content-export --help` anyway so it's likely not worth the extra effort. With that perspective, I think this all looks great as-is 👍

We could in theory add the hammer specific docs to hammer and leave the api as it is..

would some thing like ` "Choose importable if content is to be imported via the import-export process"`. work better than ` "Choose importable if content is to be imported via hammer content-import."`